release notes/v0.46.0.md

```diff
@@ -35,6 +47,38 @@ This release supports the new integration between k6 and Tempo in the cloud. Gra
 
 The new cloud traces will work "out of the box" for `k6 cloud` runs. In case of `k6 run` execution, the `K6_CLOUD_TRACES_ENABLED` environment variable has to be set to `true`.
 
+### Automatic Browser Lifecycle Handling [browser#910](https://github.com/grafana/xk6-browser/pull/910), [browser#944](https://github.com/grafana/xk6-browser/pull/944)
```

Comment:
In the breaking section I added the entry for https://github.com/grafana/xk6-browser/pull/910, [here](https://github.com/grafana/k6/blob/86e3f0bfeb8b84eee51a5c3640de4e6ce73d0449/release%20notes/v0.46.0.md?plain=1#L17), which already explains the breaking changes in the API. But then I think we need to explain better what is the current API and how the browser lifecycle is handled, so I decided to include an entry in `New features` which, as per the format defined, seems to be the section that allows for further explanation, and include here grafana/xk6-browser#944 which, after the braking changes done in 910, it includes a new functionality to handle browser lifecycle based on k6 events. Does that make sense?

Reply:
Yeah, I see what you've done, but as a user I think I'd like to know how to work with the new API and how to migrate to it. It is true that abstracting away some of the boiler plate is an improvement, but I wouldn't class it as a new feature either since it doesn't add anything new that a user couldn't do earlier.

We've also classed the browser module as a breaking change by increasing the major version, so I think that warrants a detailed explanation in the breaking changes section. I still feel that a single sub section under breaking changes where we have all the associated issues and an explanation would be a better place.

Reply:
This does not seem to fit the format defined in the template, but is similar to my suggestion in this [comment](https://github.com/grafana/k6/pull/3260#discussion_r1289715563), so let's wait for k6 core team's feedback in that discussion.

Reply:
True, but how do we best explain a major version change?

Reply:
I'm working on a proposal for the changes, bear with me.

Reply:
I have the same feeling as @ankur22. It isn't introducing something new but just changing the API interface. If we have also a new part of the API then I'm fine with splitting them between breaking change and features sections but what is available right now in the text seems only a breaking change.
Of course, I don't have enough knowledge about the topic so I could be completely wrong, as I said this is mostly my feeling as a Reader.

Reply:
See a proposal for reorganization in [9349278](https://github.com/grafana/k6/pull/3260/commits/93492783540421834b0fc09d1ebc1beda27e0499), where this has been moved into the _breaking changes_ section.